src/main/java/seedu/us/among/commons/core/Messages.java

```diff
@@ -15,11 +15,18 @@
     public static final String MESSAGE_ENDPOINTS_LISTED_OVERVIEW = "%1$d API endpoints listed!";
     public static final String MESSAGE_INVALID_JSON = "The request was not performed successfully. Check"
             + " that your data is added in the correct JSON format.";
-    public static final String MESSAGE_CONNECTION_ERROR = "Connection was lost or could not be established."
-            + " Check your internet connection and endpoint URL. This could also be a problem with the server"
-            + " you are attempting to connect to.";
+    public static final String MESSAGE_CONNECTION_ERROR = "Connection could not be established. Possible sources of "
+            + "error:\n"
+            + "- Invalid URL\n"
+            + "- Server error\n"
+            + "- Internet connection\n"
+            + "- Invalid command format\n";
```

Comment:
"Possible sources of error may include but are not limited to: "

might be a better way to put this

Reply:
ok